### PR TITLE
Use tslib's PlotHistory in optuna-dashboard 

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

NA

## What does this implement/fix? Explain your changes.

Use tslib's PlotHistory in optuna-dashboard

### CODE REVIEW

File: optuna_dashboard/ts/components/GraphHistory.tsx [hunk 1]
```diff
@@ -1,373 +1,22 @@
...
-          navigate(
-            url_prefix +
-              `/studies/${studyId}/trials?numbers=${data.points[0].x}`
-          )
-        }
-      })
-      return () => {
-        // @ts-ignore
-        element.removeAllListeners("plotly_click")
-      }
```
Comment (porink0424) [inline optuna_dashboard/ts/components/GraphHistory.tsx:114]: I think this part is missing in `tslib/react/src/components/PlotHistory.tsx`.

**Review by porink0424 — CHANGES_REQUESTED** [review]
Looks almost good to me! I left just one comment.

File: optuna_dashboard/ts/components/GraphHistory.tsx (hunk 1 above)
Comment (keisuke-umezawa) [reply optuna_dashboard/ts/components/GraphHistory.tsx:114]: I missed this part. Thank you.

**Comment by keisuke-umezawa** [comment]
@porink0424 I followed your suggestion, and implemented it like we did it for TrialTable.

**Review by porink0424 — APPROVED** [review]

**Comment by porink0424** [comment]
LGTM!

**Pull request merged**